s3/src/main/java/ch/cyberduck/core/s3/S3Session.java

```diff
@@ -243,6 +244,20 @@ else if (host.getProtocol().getOAuthAuthorizationUrl() != null) {
                 client.setProviderCredentials(credentials.isAnonymousLogin() ? null :
                         new AWSSessionCredentials(credentials.getUsername(), credentials.getPassword(),
                                 credentials.getToken()));
+                // TODO remove the following snippet: sanity check that STS credentials are valid with AWS Java SDK
```

Comment:
Out of scope.

Reply:
removed snippet -> resolving discussion.